enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/DefaultControllerEnricher.java

```diff
@@ -97,10 +97,11 @@ public DefaultControllerEnricher(MavenEnricherContext buildContext) {
     @Override
     public void addMissingResources(KubernetesListBuilder builder) {
         final String name = getConfig(Config.name, MavenUtil.createDefaultResourceName(getContext().getGav().getSanitizedArtifactId()));
-        ResourceConfig config = new ResourceConfig.Builder(getConfiguration().getResource().orElse(null))
+        ResourceConfig xmlResourceConfig = getConfiguration().getResource().orElse(null);
+        ResourceConfig config = new ResourceConfig.Builder(xmlResourceConfig)
```

Comment:
if `xmlResourceConfig` is null, is it protected in `ResourceConfig.Builder`?

Reply:
umm, Could you please elaborate wdym by protected here?

Reply:
here is `ResourceConfig` builder:
https://github.com/fabric8io/fabric8-maven-plugin/blob/21b78a929a64c21ccccee279feaef8c1dea4f318/core/src/main/java/io/fabric8/maven/core/config/ResourceConfig.java#L159-L180

Reply:
Yes there a null protection, so it is fine :)